airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_41_00_002__AddActorDefinitionConfigInjection.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.currentOffsetDateTime;
+import static org.jooq.impl.DSL.foreignKey;
+import static org.jooq.impl.DSL.primaryKey;
+import static org.jooq.impl.DSL.unique;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.JSONB;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: update migration description in the class name
+public class V0_41_00_002__AddActorDefinitionConfigInjection extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_41_00_002__AddActorDefinitionConfigInjection.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    try (final DSLContext ctx = DSL.using(context.getConnection())) {
+      addConfigInjectionTable(ctx);
+    }
+  }
+
+  private static void addConfigInjectionTable(final DSLContext ctx) {
+    final Field<JSONB> jsonToInject = DSL.field("json_to_inject", SQLDataType.JSONB.nullable(false));
+    final Field<String> injectionPath = DSL.field("injection_path", SQLDataType.VARCHAR(256).nullable(false));
```

Comment:
is there a reason to use `VARCHAR(256)` instead of `TEXT`? As far as I know, there's no benefit to using a VARCHAR on postgres.

https://stackoverflow.com/a/4849030

Reply:
jooq has the following string data types: VARCHAR, CHAR, LONGVARCHAR, CLOB, NVARCHAR, NCHAR, LONGNVARCHAR, NCLOB

`VARCHAR` with a length should be converted to a `VARCHAR` postgres type while `VARCHAR` without length will probably become `TEXT` (both are used in other tables as well)

Happy to go without a length limitation here, as you say there is no good reason to restrict it.